samples/WelcomePageSample/web.config

```diff
@@ -4,6 +4,8 @@
     <handlers>
```

Comment:
Why is this file even here? It's not needed.

Reply:
OK